sdk/src/main/java/io/radar/sdk/RadarTrackingOptions.kt

```diff
@@ -302,8 +309,9 @@ data class RadarTrackingOptions(
                 val activity = if (obj.isNull(KEY_FOREGROUND_SERVICE_ACTIVITY)) null else obj.optString(KEY_FOREGROUND_SERVICE_ACTIVITY)
                 val importance = if (obj.isNull(KEY_FOREGROUND_SERVICE_IMPORTANCE)) null else obj.optInt(KEY_FOREGROUND_SERVICE_IMPORTANCE)
                 val id = if (obj.isNull(KEY_FOREGROUND_SERVICE_ID)) null else obj.optInt(KEY_FOREGROUND_SERVICE_ID)
+                val channelName = if (obj.isNull(KEY_FOREGROUND_SERVICE_CHANNEL_NAME)) null else obj.optString(KEY_FOREGROUND_SERVICE_CHANNEL_NAME)
```

Comment:
Is this the same as using the Elvis operator?

```
val channelName = obj.optString(KEY_FOREGROUND_SERVICE_CHANNEL_NAME) ?: nil
```

Reply:
The difference is that `optString` returns an empty string if the key doesn't exist, so the result would always exist.

Reply:
Ah, got it.